src/status_im/ui/components/icons/vector_icons.cljs

```diff
@@ -23,12 +23,17 @@
 (def use-def (get-class "Use"))
 (def defs (get-class "Defs"))
 
-(def icons {:icons/chats               (slurp-svg "./resources/icons/bottom/chats_gray.svg")
-            :icons/chats-active        (slurp-svg "./resources/icons/bottom/chats_active.svg")
-            :icons/contacts            (slurp-svg "./resources/icons/bottom/contacts_gray.svg")
-            :icons/contacts-active     (slurp-svg "./resources/icons/bottom/contacts_active.svg")
-            :icons/discover            (slurp-svg "./resources/icons/bottom/discover_gray.svg")
-            :icons/discover-active     (slurp-svg "./resources/icons/bottom/discover_active.svg")
+(def icons {;;NOT USED IN MVP
+            ;;:icons/chats               (slurp-svg "./resources/icons/bottom/chats_gray.svg")
```

Comment:
How about deleting comments? They will still be accessible via git history if needed.

Reply:
ok